services/order.py

```diff
@@ -0,0 +1,34 @@
+from datetime import datetime
+from typing import Any
+
+from django.db import transaction
+
+from db.models import Ticket, User, Order, MovieSession
+
+
+def create_order(tickets: list, username: str,
+                 date: datetime = None) -> Order:
+    with transaction.atomic():
+        user = User.objects.get(username=username)
+        order = Order.objects.create(
+            user=user, created_at=date if date else datetime.now()
+        )
+
+        for ticket_data in tickets:
+            movie_session = MovieSession.objects.get(
+                id=ticket_data["movie_session"]
+            )
```

Comment:
Similar to the user retrieval, consider handling the case where the `MovieSession` does not exist by catching the `MovieSession.DoesNotExist` exception.